run-kobiton-test.md

```diff
@@ -0,0 +1,171 @@
+# Run Kobiton Automation Test on Travis CI
+This guide will demonstrate how to run your test script with Kobiton on Travis CI and how to secure your secret environment variables (i.e Kobiton API Key).
+
+## Table of contents 
+  - [Prerequisites](#prerequisites)
+  - [1. Configure Travis CI with Github repository](#1-configure-travis-ci-with-github-repository)
+  - [2. Configure to run automation test on Kobiton](#2-configure-to-run-automation-test-on-kobiton)
+  - [3. Secure your Kobiton API Key when triggering a Travis CI build](#3-secure-your-kobiton-api-key-when-triggering-a-travis-ci-build)
+  - [4. Get session information through REST API](#4-get-session-information-through-REST-API)
+  - [5. (Optional) Use cache to improve Travis CI building speed](#5-optional-use-cache-to-improve-travis-ci-building-speed)
+
+## Prerequisites
+  - Kobiton account
+  - Kobiton installed (Optional unless you want to test on your real devices)
+
+## 1. Configure Travis CI with Github repository
+
+  This part will guide you on how to sync Travis CI with a Github repository. If you already know how to do it, skip this step.
+   >- For instruction on how to link Travis CI with a Github repository, follow [this guide.](https://docs.travis-ci.com/user/legacy-services-to-github-apps-migration-guide/)
+    >- In this document, we will use a Node app to run the test script. If you do not know how to integrate a Node JS app with Travis CI, follow [this tutorial.](https://docs.travis-ci.com/user/languages/javascript-with-nodejs/) 
+
+## 2. Configure to run automation test on Kobiton
+  > You can visit [our blog](https://kobiton.com/blog/automation-web-appium-kobiton-nodejs/) for details on setting up an automation test script.
+
+  1. Write an automation test script  
```

Comment:
I moved this part above the get username/api key part because this will control the flow of the document so that the user won't get confused of which language this guideline uses.